packages/components/src/Autocomplete/Autocomplete.test.tsx

```diff
@@ -4,6 +4,15 @@ import userEvent from "@testing-library/user-event";
 import { AnyOption, Autocomplete } from ".";
 import { InputTextRef } from "../InputText";
 
+// Mock popper to avoid forceUpdate causing act warnings with testing-library.
+jest.mock("@popperjs/core", () => ({
+  createPopper: () => ({
+    destroy: jest.fn(),
+    forceUpdate: jest.fn(),
+    update: jest.fn(),
+  }),
+}));
+
```

Comment:
Just curious if you chose this style over Manual Mocks -- i.e. placing a mock for this module in a global `__mocks__` folder, a la https://jestjs.io/docs/manual-mocks

Reply:
Give it a try. We don't do it commonly and I'm no Jest expert, so I can't tell if there are pitfalls... but it seems to be in line with what you're trying to achieve in this PR.